firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/GenerativeModel.kt

```diff
@@ -122,11 +123,13 @@ internal constructor(
   )
 
   /**
-   * Generates a [GenerateContentResponse] from the backend with the provided [Content].
+   * Generates new content from the input [Content] given to the model as a prompt.
    *
-   * @param prompt [Content] to send to the model.
-   * @return A [GenerateContentResponse]. Function should be called within a suspend context to
-   * properly manage concurrency.
+   * This function should be called within a suspend context to properly manage concurrency.
+   *
+   * @param prompt The input(s) given to the model as a prompt.
+   * @return The content generated by the model.
+   * @throws [FirebaseVertexAIException] if the request failed.
```

Comment:
Is this throws a bit vague? It may be better to note that it'd be one of several subclasses, even if the bulk is documented in FirebaseVertexAIException and just mentioned here. Perhaps a `@see`?